streaming/index.js

```diff
@@ -850,6 +850,18 @@ const startWorker = async (workerId) => {
     res.end('OK');
   });
 
+  app.get('/metrics', (req, res) => server.getConnections((err, count) => {
+    res.writeHeader(200, { 'Content-Type': 'text/plain' });
```

Comment:
It best to have a proper `Content-Type` here.

[The spec says](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#overall-structure):

> The content type MUST be: `application/openmetrics-text; version=1.0.0; charset=utf-8`